includes/managers/controls.php

```diff
@@ -1078,6 +1078,24 @@ public function get_teaser_template( $texts ) {
 		return ob_get_clean();
 	}
 
+	/**
+	 * Get Responsive Control Device Suffix
+	 *
+	 * @param array $control
+	 * @return string $device suffix
+	 */
+	public static function get_responsive_control_device_suffix( array $control ): string {
```

Comment:
`:string` - are we specifying return types now?

Reply:
We have officially dropped support for PHP 5.6. Since we now support PHP >7 we can do this. I believe our PHPCS even hints for it